README.md

```diff
@@ -43,35 +43,86 @@ It will create some environment variables to ease access to some folders:
+python WRemnants/scripts/combine/setupCombineWMass.py -i outputFolder/mw_with_mu_eta_pt_scetlib_dyturboCorr.hdf5 -o outputFolder/
 ```
 The input file is the output of the previous step.
-Add __--wlike__ if running a Wlike analysis.
-The default path specified with __-o__ is the local folder. Subfolder named WMass/ or ZMassWLike/ is created in that depending on the chosen analysis.
+The default path specified with __-o__ is the local folder. A subfolder with name identifying the specific analysis (e.g. WMass_pt_eta/) is automatically created inside it. Some options may add tags to the folder name: for example, using --doStatOnly will  call the folder WMass_pt_eta_statOnly/.
  
 Combine the datacards for single charges and run the fit (Asimov only)
 ```
-python WRemnants/scripts/combine/fitManager.py -i WMass/smoothSF/muonCorr_none/scetlibCorr_nnpdf31/byHelicityPtCharge/ --comb --skip-fit-data
+python WRemnants/scripts/combine/fitManager.py -i outputFolder/WMass_pt_eta/ --comb --skip-fit-data
 ```
 Run the fits for single charges (Asimov only). These can be produced in the same output folder as the combination, since a postfix is automatically appended to the output card and fit result files.
 ```
-python WRemnants/scripts/combine/fitManager.py -i WMass/smoothSF/muonCorr_none/scetlibCorr_nnpdf31/byHelicityPtCharge/ --fit-single-charge --skip-fit-data [-c plus|minus|both]
+python WRemnants/scripts/combine/fitManager.py -i outputFolder/WMass_pt_eta/ --fit-single-charge --skip-fit-data [-c plus|minus|both]
 ```
 
-**NOTE**: to run __fitManager.py__ one has to set a Centos 7 environment with __cmsssw-cc7__. Then, one has to activate __cmsenv__ from the folder where combine is installed (once the environment is set one can keep working from inside WRemnants).
+**NOTE**:
+ * to run __fitManager.py__ one has to set a Centos 7 environment with __cmsssw-cc7__. Then, one has to activate __cmsenv__ from the folder where combine is installed (once the environment is set one can keep working from inside WRemnants).
+ * Each script has tons of options, to customize a gazillion of things, it's simpler to learn them by asking an expert rather that having an incomplete summary here (developments happen faster than documentation anyway).
+
+### Making plots
 
-Make plots on prefit histograms (output root file from __setupCombineWMass.py__) or the fit results (output of __fitManager.py__)
+There are many scripts to do every kind of plotting, and different people may have their own ones. We'll try to put a minimal list with examples here ASAP.
+
+Plot Wmass histograms from hdf5 file (from Wmass histmaker) in the 4 iso-MT regions (can choose only some). It also makes some plots for fakes depending on the chosen region. It is also possible to select some specific processes to put in the plots.
```

Comment:
imo it would make a little more sense to keep the main readme shorter and link to other readme in the relevant directories for things like plotting

Reply:
Good point, I will edit this readme to point to other specific readme files stored in the plotting folders